assets/dev/scss/editor/_global.scss

```diff
@@ -166,3 +165,31 @@
 i.eicon-tilted {
 	transform: rotate(90deg);
 }
+
+.elementor-control .elementor-control-content.elementor-control-media-size {
+	margin-top: 15px;
+}
+
+.elementor-control-media-size .elementor-control-field {
+	padding: 0px;
+}
+
+.elementor-control-media-size .elementor-control-responsive-switchers {
+	display: none;
+}
+
+.elementor-control-media-display-size.select {
+	padding-bottom: 0px;
+	width: 72%;
```

Comment:
what's this magic number?

Reply:
The panel has x Width, the select need to be 72% of his wrapper width in order to be aligned with all the other inputs.

Reply:
can you fix it with flex or something?

Reply:
No, because he won't be aligned with all other inputs below him. (I test it)